pkg/plugins/optional/helm/v1alpha/scaffolds/internal/templates/chart-templates/helpers_tpl.go

```diff
@@ -28,12 +28,15 @@ var _ machinery.Template = &HelmHelpers{}
 type HelmHelpers struct {
 	machinery.TemplateMixin
 	machinery.ProjectNameMixin
+
+	// Directory is the directory where the Helm chart will be scaffolded
+	Directory string
```

Comment:
I've also thought about adding a new `machinery.HelmMixin` which could be used for all helm-based structs, but decided to not do it for now as i wanted your feedback on this first.